demo/index.js

```diff
@@ -2,25 +2,27 @@ import { createElement, render, Component, Fragment } from 'preact';
 // import renderToString from 'preact-render-to-string';
 import './style.scss';
 import { Router, Link } from 'preact-router';
-import Pythagoras from './pythagoras';
-import Spiral from './spiral';
-import Reorder from './reorder';
-import Todo from './todo';
-import Fragments from './fragments';
-import Context from './context';
+// import Pythagoras from './pythagoras';
```

Comment:
Uh oh, are these needed to be commented out? Were they messing up with your demo?

Reply:
Whoops! I commented these out because debugging the 7MB demo bundle brought the my laptop's Chrome DevTools to a standstill (not entirely but it was very difficult to debug). So I commented out the other demos to make it easier to debug. I'll add them back.

In other PR I'd like to code-split all these demos behind lazy and Suspense, but we need a functioning Suspense component first 😜

Reply:
Removed the comments